lib/players/computer.rb

```diff
@@ -0,0 +1,25 @@
+module Players
+
+  class Computer < Player
+
+    # HMMMM!
+
+    # simple logic:
+
+    # def move(board)
+    #   "1"
+    # end
+
+    def open_positions(board)
+      @open = []
+      board.cells.each_with_index { |cell, index| @open << (index + 1).to_s if cell == " "}
+      @open
+    end
+
+    def move(board)
+      positions = open_positions(board)
+      @move = positions[0]
+      @move
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.